posthog/tasks/scheduled.py

```diff
@@ -223,6 +223,8 @@ def setup_periodic_tasks(sender: Celery, **kwargs: Any) -> None:
         name="process scheduled changes",
     )
 
+    add_periodic_task_with_expiry(sender, 3600, invalid_web_replays.s())
```

Comment:
moving it within the file and removing the overridden name to try and shake the task loose